BootloaderCorePkg/Tools/PrepareBuildComponentBin.py

```diff
@@ -246,7 +257,15 @@ def Main():
         CopyBins (fsp_repo_dir, sbl_dir, fsp_inf)
 
     microcode_inf = os.path.join(sbl_dir, 'Silicon', silicon_pkg_name, 'Microcode', 'Microcode.inf')
-    CopyBins (ucode_repo_dir, sbl_dir, microcode_inf)
+    microcode_bins = CopyBins (ucode_repo_dir, sbl_dir, microcode_inf)
+
+    # Add microcode padding
+    if microcode_bins != None and len(sys.argv) > 4:
+        slot_size = int(sys.argv)
+        for microcode_bin in microcode_bins:
+            microcode_bin = os.path.join(sbl_dir,microcode_bin[1])
+            DoAddPaddingData (microcode_bin, slot_size)
```

Comment:
Does DoAddPaddingData gets invoked for slot_size zero cases also?

Reply:
Yes, but in such a case, it does not add any padding bytes.